src/libgeodecomp/io/logger.h

```diff
@@ -18,6 +18,10 @@ namespace LibGeoDecomp {
  * happens at compile time, thus no runtime overhead is incurred for
  * filtered messages.
  */
+#ifdef ERROR
```

Comment:
Quick question: what's this good for? We're not using ERROR as a debug level, right?